src/analyses/goto_check.cpp

```diff
@@ -991,7 +991,8 @@ void goto_checkt::pointer_validity_check(
       allocs=disjunction(disjuncts);
     }
 
-    if(flags.is_unknown() || flags.is_null())
+    if(flags.is_unknown() ||
```

Comment:
Point of curiosity: all of these `is_` tests are `may_be_` tests, right?

Reply:
Yes, indeed.